www/src/pages/newsletter.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react"
+import Helmet from "react-helmet"
+import { rhythm } from "../utils/typography"
+
+import Layout from "../components/layout"
+import Container from "../components/container"
+import EmailCaptureForm from "../components/email-capture-form"
+
+class NewsLetter extends Component {
+  render() {
+    const { location, data } = this.props
+
+    return (
+      <Layout location={location}>
+        <Helmet>
+          <title>Newsletter</title>
+        </Helmet>
+        <Container
+          hasSideBar={false}
+          css={{
+            display: "flex",
+            flexDirection: "column",
+          }}
+        >
+          <h1 id="introduction" style={{ marginTop: 0 }}>
+            NewsLetter
+          </h1>
+          <div>
+            Sign up for the Gatsby newsletter to keep up with the latest news
```

Comment:
Keep up with the latest from the Gatsby community! Hear about new features, tips & tricks, and what people are building.

Reply:
Thanks for the blurb. That sounds much nicer